HW8/marian_hrychak/task_1/main.py

```
@@ -0,0 +1,4 @@
+from utils import *
```

Ruff (0.8.2)

`from utils import *` used; unable to detect undefined names

(F403)